bin/update-dockerfiles.sh

```diff
@@ -5,6 +5,8 @@ SED="sed -i"
 sed --version 2>/dev/null || SED="sed -i.bkp"
 
 LTS_VERSIONS=("8" "11" "17" "21")
+AL2_VERSIONS=("8" "11" "17" "21")
```

Comment:
Can you rename this to AL2_GENERIC_LINUX_VERSIONS? Other option would be to remove this and change the check on line 57 to see if the file exists, like we do for al2023-generic.